### DISCUSSION TITLE
AutoIPTV-Hotel直播源

### DISCUSSION THREAD

**Discussion by shit6** [HerbertHe/iptv-sources#91] [Ideas]
https://github.com/cymz6/AutoIPTV-Hotel
能否把这个加上？

**Comment by leiyou-li** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by shit6** [comment]
还有1个https://github.com/BurningC4/Chinese-IPTV

**Reply by HerbertHe** [reply]
> 还有1个https://github.com/BurningC4/Chinese-IPTV

这个更新频率挺低的，暂时不考虑

**Comment by HerbertHe** [comment]
> https://github.com/cymz6/AutoIPTV-Hotel 能否把这个加上？

这个等我有时间处理一下

**Comment by shit6** [comment]
感谢大佬回复，这个源ipv4也可以看
&nbsp;


shit6
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Herbert ***@***.***&gt;; 
发送时间: 2024年6月4日(星期二) 下午4:41
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [HerbertHe/iptv-sources] AutoIPTV-Hotel直播源 (Discussion #91)














  
https://github.com/cymz6/AutoIPTV-Hotel 能否把这个加上？
  
这个等我有时间处理一下

**Comment by HerbertHe** [comment]
@shit6 https://github.com/HerbertHe/iptv-sources/commit/ddcd240d41802b946d5e6c1ffad56f8da870a218
